Lib/pydoc.py

```diff
@@ -1775,10 +1776,15 @@ def render_doc(thing, title='Python Library Documentation: %s', forceload=0,
     return title % desc + '\n\n' + renderer.document(object, name)
 
 def doc(thing, title='Python Library Documentation: %s', forceload=0,
-        output=None):
+        output=None, is_cli=False):
     """Display text documentation, given an object or a path to an object."""
     if output is None:
-        pager(render_doc(thing, title, forceload))
+        try:
+            pager(render_doc(thing, title, forceload))
+        except ImportError:
+            if is_cli:
+                raise ImportError(missing_pattern % thing) from None
```

Comment:
Why not re-raising `ImportError` that is raised in `resolve` called in `render_doc` instead of constructing the same exception again?

Reply:
(I am not a core dev to review though, just trying to help. Maybe you should ping someone who is responsible for this code)

Reply:
Thanks, @chgnrdv !
Yeah, that's look right, i'll soon change this.

Just FYI, when you reviewing someone `PR`, on the page `Files changed` you can click a button called `Add a suggestion`. This more convenient way in this case. =)

Reply:
Thank you, this is really helpful. I saw this thing being done by reviewers and wondered how to do it myself :-)

Your code looks good to me, but you have to wait for someone who is authorized for review. This might take some time since the number of contributors far exceeds the number of reviewers.